avocado_vt/plugins/vt_list.py

```diff
@@ -105,12 +105,12 @@ def configure(self, parser):
         vt_compat_group_lister.add_argument("--vt-list-guests",
                                             action="store_true",
                                             default=False,
-                                            help="Also list the available "
+                                            help="List the available "
                                             "guests (this option ignores the "
                                             "--vt-config and --vt-guest-os)")
         vt_compat_group_lister.add_argument("--vt-list-archs", default=False,
                                             action="store_true",
-                                            help="Also list the available "
+                                            help="List the available "
```

Comment:
I think we can skip this one.  It's one of those operations that are not run all the time, so the extra time won't hurt that bad.  Then we keep the current behavior, at least a little longer.  Although this is low priority, it keeps pointing towards individual commands IMHO.